kdc/kerberos5.c

```diff
@@ -38,6 +38,41 @@
 #undef __attribute__
 #define __attribute__(X)
 
+struct kdc_pa_auth_status {
+    int auth_status;
+    const char *auth_details;
+    void *free_ptr;
+};
+
+static krb5_error_code
+_kdc_audit_auth_status(astgs_request_t r,
+		       int auth_status,
+		       const char *auth_details,
+		       const char *pa_type)
+{
+    krb5_kdc_configuration *config = r->config;
+    struct HDB *hdb;
+    int ret = 0;
```

Comment:
Sorry to keep finding things. `ret` should be `krb5_error_code`

`r->config` can be accessed directly without a temporary variable

Reply:
Done.